test/sqlite.spec.js

```diff
@@ -163,7 +163,7 @@ describe('sqlite', () => {
     sql = ' CREATE TABLE `Test` (  `id` int(11) NOT NULL,  `name` varchar(255) COLLATE utf8mb4_unicode_520_ci NOT NULL,  PRIMARY KEY (`id`),  UNIQUE KEY `name` (`name`));'
     ast = parser.astify(sql, { database: 'mariadb' })
     expect(parser.sqlify(ast, DEFAULT_OPT)).to.be.equal('CREATE TABLE "Test" ("id" INT(11) NOT NULL, "name" VARCHAR(255) NOT NULL, PRIMARY KEY ("id"), UNIQUE ("name"))')
-    sql = `SELECT 
+    sql = `SELECT
```

Comment:
This is from the `trim_trailing_whitespace` editor rule, I can back it out if you prefer formatting fixes to be in separate PRs.